extensions/amp-ad/0.1/test/test-concurrent-load.js

```diff
@@ -75,13 +75,27 @@ describes.realWin('concurrent-load', {}, env => {
       installTimerService(win);
     });
 
-    it('should throttle ad loading one per second', () => {
+    it('should throttle ad loading one per second', function* () {
       expect(is3pThrottled(win)).to.be.false;
       incrementLoadingAds(win);
       expect(is3pThrottled(win)).to.be.true;
       clock.tick(999);
+      yield;
```

Comment:
Super cool, just learned about generators and `yield`. 😄 

Could `await` and Promises could achieve the same goal (but perhaps read more naturally)? E.g. in the test below:
```
let loadAds = incrementLoadingAds(win); // returns Promise
expect(is3pThrottled(win)).to.be.true;
await loadAds; // replaces invocation of resolver and `yield`
expect(is3pThrottled(win)).to.be.false;
```